pages/api/performance/collect.js

```diff
@@ -0,0 +1,27 @@
+import prisma from '@prisma/index'; // Prisma client
+import { getSession } from 'next-auth/client'; // Session handling
+
+// get performances by specifying the corresponding eventID
+// POST with { eventID: id }
+export default async (req, res) => {
+  // Collect session from request
+  const session = await getSession({ req });
+
+  const { event_id: eventID } = req.body;
```

Comment:
can't we just do ```const { event_id } = req.body```?

Reply:
Also, are we deciding to get query stuff from ```req.body``` instead of ```req.query```?

Reply:
Yes. Let's use JSON body :)

Reply:
@thepenguinco Let me know if we decide to go with req.body or req.query for GET requests. I'll update my PR accordingly